Gemfile

```diff
@@ -155,8 +155,8 @@ group :development do
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem "spring"
+  gem "spring-commands-rspec"
```

Comment:
Is this related to this change? Is there a `bin/rspec` that we're missing?

Reply:
👍 no it's not but I'll add those in a separate PR. Will add those in a separate PR.